.github/workflows/manual-publish.yml

```diff
@@ -34,3 +36,17 @@ jobs:
         uses: ./.github/actions/publish
```

Comment:
I think you need to set the id of the step here to publish.

Reply:
Not sure I'm understanding this comment - `id: publish` is set in line 34(?)

Reply:
🤦🏼 just didn't see it since it was collapsed in the UI.